### PR TITLE
Feature/erssup 56401

### PR DESCRIPTION
**Pull request by nhsd-jack-wainwright**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Review by nhsd-hubert-raich — APPROVED** [review]

**Review by adamoldfield-nhs — APPROVED** [review]
I have done a high level review. Mainly looking at the generated documentation output at https://uat.nhsd.io/developer/api-catalogue/named/jack-wainwright. Please have another develop do a lower level review of the OAS file and sandbox implementation.

File: specification/components/schemas/responses/searchForService/400Response.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  | Error code               | Description                                                         |
+  | ------------------------ | ------------------------------------------------------------------- |
+  | MISSING_VALUE            | A field defined as mandatory for an endpoint has not been provided. |
+headers:
+  x-correlation-id:
+    $ref: '../../headers/response/CorrelationID.yaml'
+  x-request-id:
+    $ref: '../../headers/response/RequestID.yaml'
+  Etag:
```
[the remark this answers is not in the document]
Comment (nhsd-jack-wainwright) [reply specification/components/schemas/responses/searchForService/400Response.yaml:13]: You're right yup, have removed this now

**Comment by nhsd-jack-wainwright** [comment]
This branch will now be converted into a shared GPIT-Futures branch which will be used to house ongoing OAS file changes for the up-coming GPIT-Futures work. Due to this these changes will be merged later when the GPIT-Futures branch is merged, so this PR can now be closed

**Pull request closed without merging**